specifyweb/specify/migrations/0002_geo.py

```diff
@@ -0,0 +1,197 @@
+# Generated by Django 3.2.15 on 2024-06-21 21:56
+# TODO: Finish editing this migration for the new app structure
+
+from django.db import migrations, models
+from django.db.models import Subquery, OuterRef
+import django.db.models.deletion
+import django.utils.timezone
+import specifyweb.specify.models
+from specifyweb.specify.models import (
+    Taxontreedef,
+    Taxontreedefitem,
+    Discipline,
+    Collectionobject,
+    CollectionObjectType,
+    Collection,
+)
+
+# Migrations Operations Order:
+# 1. Create CollectionObjectType
+# 2. Add CollectionObjectType (coType) realtionship to CollectionObject
+# 3. Add hasreferencecatalognumber field to CollectionObject
+# 4. Create default collection object types based on each discipline
+# 5. Create new default taxon trees and ranks
+# 6. Create CollectionObjectGroup
+# 7. Create CollectionObjectGroupJoin
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        ('specify', '0001_initial'),
+    ]
+
+    def add_new_default_taxon_trees(apps, schema_editor):
+        TAXON_TREES = ["Mineral", "Rock", "Meteorite", "Fossil"]
```

Comment:
Just for the record, this is a placeholder. Users will be able to define their own tree and this migration _should not_ create trees in databases that do not need them.